test-projects/gts/package.json

```diff
@@ -2,14 +2,22 @@
   "name": "@test-project/gts",
   "private": true,
   "scripts": {
-    "test": "eslint ."
+    "test": "eslint .",
+    "types": "tsc --noEmit"
   },
   "devDependencies": {
+    "@ember/test-waiters": "^3.1.0",
+    "@glint/template": "^1.3.0",
+    "@glimmer/tracking": "^1.1.2",
+    "@glimmer/component": "^1.1.2",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
     "ember-eslint-parser": "workspace:^",
+    "ember-source": "^5.6.0",
     "eslint": "^8.0.1",
     "eslint-plugin-ember": "^12.0.0",
+    "limber": "^0.0.1",
```

Comment:
This isn't related, we should remove this dependency. Last publish was 10 years ago, too

Reply:
well, it helped reduce some of the `any` types coming from limber imports

Reply:
mm, maybe we do not need it